person.rb

```diff
@@ -0,0 +1,21 @@
+# person.rb
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] It is advisable to use a generated ID, rather than requesting it as a parameter in your `initialize method`, kindly see the code snippet below to guide you on how to do that :+1: :sparkles: 
- code snippet
```ruby
 @id = Random.rand(1..1000)
````